src/repos/MatchingRepo.ts

```diff
@@ -0,0 +1,77 @@
+import { getConnectionManager, Repository } from 'typeorm';
+import Constants from '../common/constants';
+import DaySchedule from '../entities/DaySchedule';
+import Matching from '../entities/Matching';
+import Time from '../entities/Time';
+import User from '../entities/User';
+
+const dayScheduleDB = (): Repository<DaySchedule> => getConnectionManager().get().getRepository(DaySchedule);
+const matchingDB = (): Repository<Matching> => getConnectionManager().get().getRepository(Matching);
+const timeDB = (): Repository<Time> => getConnectionManager().get().getRepository(Time);
+const userDB = (): Repository<User> => getConnectionManager().get().getRepository(User);
+
+const createTime = async (
+  time: number
+): Promise<void> => {
+  if (!(Constants.VALID_TIMES.includes(time))) {
+    throw Error('Invalid time');
```

Comment:
could this be more descriptive like why is it an invalid time -- maybe print what time the user tried to create?